### DISCUSSION TITLE
latest?

### DISCUSSION THREAD

**Discussion by DirektX** [Rem0o/FanControl.Releases#1076] [General]
Latest release is 122:
https://github.com/Rem0o/FanControl.Releases/releases/latest
Am I wrong?

**Comment by Impact123** [comment]
Currently the latest version is `126`. You can run `Updater.exe` or [download it from here](https://github.com/Rem0o/FanControl.Releases/blob/master/FanControl.zip).
@Rem0o Maybe you could look into a github workflow that, at least partially, automates publishing releases for you.

**Reply by DirektX** [reply]
Please be gentle for me.
I dont understand why this link shows version 122 and not 126:
https://github.com/Rem0o/FanControl.Releases/releases/latest
Thank you

**Reply by Impact123** [reply]
Because they haven't released the version there yet. It's just out of sync with the files in the repository that's all.

**Reply by Rem0o** [reply]
I only make the release when the version has been out a few days without issues.
